app/_src/gateway/support/sbom.md

```diff
@@ -5,10 +5,11 @@ toc: false
 
 A software bill of materials (SBOM) is an inventory of all software components (proprietary and open source), open source licenses, and dependencies in a given product. A software bill of materials (SBOM) provides visibility into the software supply chain and any license compliance, security, and quality risks that may exist.
 
-Starting in {{site.ee_product_name}} 3.3, we are generating SBOMs for our artifact images.
-You can learn more about this from our software bill of materials knowledge base page:
-* [{{site.base_gateway}} 3.3.0.0 SBOM](https://support.konghq.com/support/s/article/SBOM-Artifacts-for-Kong-Gateway-3-3-0-0).
-* [{{site.base_gateway}} 3.4.0.0 SBOM](https://support.konghq.com/support/s/article/SBOM-Artifacts-for-Kong-Gateway-3-4-0-0)
-* [{{site.base_gateway}} 3.5.0.0 SBOM](https://support.konghq.com/support/s/article/SBOM-Artifacts-for-Kong-Gateway-3-5-0-0).
-* [{{site.base_gateway}} 3.6.0.0 SBOM](https://support.konghq.com/support/s/article/SBOM-Artifacts-for-Kong-Gateway-3-6-0-0).
-* [{{site.base_gateway}} 3.7.0.0 SBOM](https://support.konghq.com/support/s/article/SBOM-Artifacts-for-Kong-Gateway-3-7-0-0).
+Starting in {{site.ee_product_name}} 3.3, we are generating SBOMs for our artifact images:
```

Comment:
```suggestion
Starting in {{site.ee_product_name}} 3.3, we are generating SBOMs for our artifact images.
See the following links to directly download the SBOM for your {{site.ee_product_name}} version:
```
Something like this? Would be good to warn people that they're about to download something.